request-processor/docker-entrypoint.sh

```
@@ -1,4 +1,4 @@
 #!/bin/sh
 set -e
 echo "${DATABASE_URL}"
-celery --config celeryconfig -A tasks worker -l  INFO  -P eventlet --concurrency ${CELERY_WORKER_CONCURRENCY:-1}
+celery --config celeryconfig -A tasks worker -l  INFO  -P prefork --concurrency ${CELERY_WORKER_CONCURRENCY:-1}
```

Shellcheck (0.11.0)

[info] Double quote to prevent globbing and word splitting.

(SC2086)